third_party/2and3/redis/client.pyi

```diff
@@ -274,7 +274,7 @@ class Redis(object):
     def hincrbyfloat(self, name: _Key, key: _Key, amount: float = ...) -> float: ...
     def hkeys(self, name: _Key) -> List[bytes]: ...
     def hlen(self, name: _Key) -> int: ...
-    def hset(self, name: _Key, key: _Key, value: _Value) -> int: ...
+    def hset(self, name: _Key, key: _Key, value: _Value, mapping: Mapping[_Value, _Value]) -> int: ...
```

Comment:
Should be `Optional`, since it accepts `None` as value, and have a default value:

```suggestion
    def hset(self, name: _Key, key: _Key, value: _Value, mapping: Optional[Mapping[_Value, _Value]] = ...) -> int: ...
```

Reply:
Thank you!